daemon/codec.c

```diff
@@ -2911,6 +2911,10 @@ static void __dtx_send_later(struct codec_timer *ct) {
 	while (true) {
 		// do we have a packet?
 		dtxp = g_queue_peek_head(&dtxb->packets);
+	  if (dtxp && !dtxp->packet) {
+		  g_queue_pop_head(&dtxb->packets);
+      continue;
+    }
```

Comment:
I'm not sure this is right. AFAICT the entry with the NULL packet still needs to be processed.

Reply:
sure, this PR is just for info to show potential null ptr deref due to null packets in dtx buffer.

since fix for #1485 is sort of incomplete

Reply:
@rfuchs in particular 2nd  commit in this pr (eb8ee2763732f73e825f2101980b2509611152c3) shows that __dtx_drift_shift and __dtx_drift_drop can step on nullptr in similar way

Reply:
Yeah I get that, but I'm saying that simply discarding the entry upon null ->packet is the wrong approach.